docs/30-customize-cluster.md

```diff
@@ -1,19 +1,122 @@
+      "max": 0
+    },
+    "lowPriorityNodes": {
+      "min": 1,
+      "max": 1
+    },
+    "autoscaleFormula": "QUEUE"
+  },
+  "containerImage": "rocker/tidyverse:3.4.1",
+  "rPackages": {
+    "cran": [],
+    "github": [],
+    "bioconductor": [],
+    "githubAuthenticationToken": ""
+  },
+  "commandLine": []
+}
+```
+
+Note: \_If no 'containerImage' property is set, rocker/tidyverse:latest will be used. This usually points to one of the latest versions of R.\_
+
+### Building your own container
+
+Building your own container gives you the flexibility to package any specific requirements, packages or data you require for running your workloads. We recommend using a debian based OS such as debian or ubuntu to build your containers and pointing to where R is in the final CMD command. For example:
+
+```dockerfile
+FROM debian:stretch
+...
+CMD ["/usr/bin/R"]
+```
+
+Or alternitavely,
+
+```dockerfile
+FROM ubuntu:16.04
+...
+CMD ["R"]
+```
+
+There is no requirement to be debian based. For consistence with other pacakges it is recommeneded though. Please note though that the container **must be based off a Linux distribution as Windows is not supported**.
+
+### List of tested container images
+
+The following containers were tested and hopefully cover the most common cases for end users.
+
+Container Image | R type | Description
+--- | --- | ---
+[rocker/tidyverse](https://hub.docker.com/r/rocker/r-ver/) | Open source R | Tidyverse is provided by the rocker org and uses a standard version of R developed by the open soruce community. rocker/tidyverse typically keeps up with the latest releases or R quite quickly and has versions back to R 3.1
+[nuest/mro](https://hub.docker.com/r/nuest/mro/) | Microsoft R Open | [Microsoft R Open](https://mran.microsoft.com/open/) is an open source SKU of R that provides out of the box support for math packages, version pacakge support with MRAN and improved performance over standard Open Source R.
+
+* We recommend reading the details of each package before using it to make sure you understand any limitaions or requirements of using the container images.
+
+## Running Commands when the Cluster Starts
 
 The commandline property in the cluster configuration file allows users to prepare the nodes' environments. For example, users can perform actions such as installing applications that your foreach loop requires.
 
-Note: Batch clusters are run with Centos-OS Azure DSVMs.
+Note: Batch clusters are provisioned with Ubuntu 16.04.
+
+Note: All commands are already run as the sudo user, so there is no need to append sudo to your command line. \_Commands may fail if you add the sudo user as part of the command.\_
+
+Note: All commands are run on the host node, not from within the container. This provides the most flexibility but also requires a bit of understanding on how to run code from within R and how to load directories correctly. See below for exposed environement variables, directories and examples.
```

Comment:
shouldn't the commands be run in the container?

Reply:
i can actually see a scenario for both

Reply:
@jiata - Yeah, we need to support both. Especially with this new user who has lots of special requirements, I think we need to consider having two command line options, one on the host and one on the container.

Reply:
yeah makes sense. i'm thinking probably just get this merged in first and have custom script on containers as another feature, right?